app/models/comment.rb

```diff
@@ -1,2 +1,8 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ] The same here, It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* `after_save` to call the `#comments_posts_counter` method every time a new like has been saved to the database, then make the `comments_posts_counter` method to be a `private` method 👍🏻 💯 
- 🤔  Your code should look like this 👇🏻 :

```ruby
# app/models/comment.rb
...
     after_save : comments_posts_counter
   
     private

    def comments_posts_counter
       post.increment!(:likes_counter)
    end
```